config/src/config/api_config.rs

```diff
@@ -53,6 +53,8 @@ pub struct ApiConfig {
     pub max_submit_transaction_batch_size: usize,
     /// Maximum page size for transaction paginated APIs
     pub max_transactions_page_size: u16,
+    /// Maximum page size for block transaction APIs
+    pub max_block_transactions_page_size: u16,
```

Comment:
Call me crazy but is u16 too small? I can imagine a world where MAX_SENDING_BLOCK_TXNS is larger than u16 max. Maybe either make this u32 or cast MAX_SENDING_BLOCK_TXNS to u16 safely (ceil).